files/justfiles/utilities.just

```diff
@@ -152,14 +152,22 @@ enable-flathub-unfiltered:
     #!/usr/bin/bash
     flatpak remote-add --if-not-exists --user flathub https://flathub.org/repo/flathub.flatpakrepo
 
+# search for an installed SELinux module
+semodule-check MODULE:
+    #! /bin/run0 /bin/bash
+    if [ -n "$( semodule -l | grep {{ MODULE }} )" ] ; then
```

Comment:
does this work properly? as in, does this have greppable output?

Reply:
As long as its run in a root shell, yes, `semodule -l` outputs a list which can be grepped.